ArcGISRuntimeSDKQt_CppSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.cpp

```diff
@@ -137,8 +134,9 @@ void DeleteFeaturesFeatureService::connectSignals()
     {
       Feature* feat = iter.next();
       // emit signal for QML
-      m_featureType = feat->attributes()->attributeValue("typdamage").toString();
-      emit featureTypeChanged();
+      QString featureType = feat->attributes()->attributeValue("typdamage").toString();
+      m_mapView->calloutData()->setTitle("<br><b><font size=\"+2\">" + featureType + "</font></b>");
```

Comment:
this could also probably do with a comment to explain why we are using HTML tags